ride_share.rb

```diff
@@ -0,0 +1,167 @@
+drivers = [
+  { 
+    :DR0001  =>
```

Comment:
It's probably more standard to use this syntax

```suggestion
    DR0001:
```